lib/CMM/providers/is_cmm_admin_provider.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+import 'package:myecl/user/providers/user_provider.dart';
+
+final isCMMAdminProvider = StateProvider<bool>((ref) {
+  final me = ref.watch(userProvider);
+  for (final group in me.groups) {
+    if (group.name == "CMM") {
```

Comment:
You better have to check with the group.id instead of the name, because nothing is preventing another group to be named CMM, thus giving its members the visual of the admin panel, even if Hyperion will block their requests